desktop/main/injectFilesToOpen.ts

```
@@ -35,8 +38,12 @@ export default async function injectFilesToOpen(
       files: filesToOpen,
     });
 
+    log.debug(`Set input files #${inputElementId}: ${filesToOpen.join(",")}`);
+
     // clear the files once we've opened them
     filesToOpen.splice(0, filesToOpen.length);
+  } catch (err) {
+    log.error(err);
```

should we pop up an alert box so the user knows something went wrong instead of a silent failure?

---

Using this? https://www.electronjs.org/docs/latest/api/dialog

---

yeah something like that. Since the user probably expected something to happen if they opened a file.